src/aleph/handlers/content/store.py

```diff
@@ -175,7 +177,7 @@ async def fetch_related_content(
                 )
                 do_standard_lookup = True
 
-        if do_standard_lookup:
+        if config.storage.store_files.value and do_standard_lookup:
```

Comment:
Is it relevant to still register the file in the database with `upsert_file` in this case or should that be omitted as well ? :thinking:

Reply:
The instructions that I got is that the current behavior doesn't mark the message as acknowledge which we don't want so I believe that registering this in the db would do it?

But the code is really unclear and not documented so I'm quite confuse on how to achieve that :/